applications/ws-irc3sp/public/v1/irc3sp.ini

```diff
@@ -18,16 +18,18 @@ post.parameters.1.description = Indent or not the JSON Result
 plugin = @ezs/spawn
 # JSONParse
 plugin = @ezs/basics
+# sentences
+plugin = ./v1/local.js
 
 [JSONParse]
 legacy = false
```

Comment:
@parmentf fast but dangerous...

Reply:
Dangerous ?
Because of the lack of testing ?
Sure.

Reply:
bad structure causes a crash cf. https://github.com/tsouza/yajs/issues/5